api/src/main/resources/liquibase.xml

```diff
@@ -224,4 +224,11 @@
             UPDATE global_property SET property_value = 'true' WHERE property ='dataentrystatistics.started';
         </sql>
     </changeSet>
+
+    <changeSet id="ugandaemr-25072019-1409" author="slubwama">
```

Comment:
@slubwama the short name, and description also needs to be updated 
<img width="889" alt="Screenshot 2019-07-25 14 52 35" src="https://user-images.githubusercontent.com/689900/61872444-f73c9100-aeeb-11e9-9682-4393ef107ec6.png">